packages/expo-secure-store/android/src/main/java/expo/modules/securestore/SecureStoreModule.java

```diff
@@ -466,9 +470,9 @@ public String decryptItem(JSONObject encryptedItem, KeyStore.SecretKeyEntry secr
 
     protected Context mContext;
     private AESEncrypter mAESEncrypter;
-    protected SecureRandom mSecureRandom;
+    /* package */ SecureRandom mSecureRandom;
```

Comment:
If something outside this class needs to access it, it should call `getSecureRandom()`

Reply:
Changed to `private`, nothing outside of this class accesses it.